CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Fixed
 * Items.Item \<index> <- value gives wrong code. [#2498](https://github.com/fsprojects/fantomas/issues/2498)
+Fix invalid output for record with accessibility modifier & members [2516](https://github.com/fsprojects/fantomas/pull/2516)
```

Comment:
Please don't use the PR link but the issue title and link instead.
Also, start the entry with `*`.